ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/91521_SCAPP_apply-to-become/91489_Apply-to-become-get-application-data-map.js

```diff
@@ -0,0 +1,402 @@
+            expect(userData.data.applyingSchools[0].schoolPartOfLaReorganizationPlan).to.eq(true)
+        })
+        cy.get('[test-id="Further_information_Additional_details5_key"]').should('contain.text', 'Is the school part of any local authority closure plans?')
+        cy.get('[test-id="Further_information_Additional_details5_value"]').should('contain.text', 'Yes')
+        cy.fixture('cath121-body.json').as('userData').then((userData) => {
+            expect(userData.data.applyingSchools[0].schoolPartOfLaClosurePlan).to.eq(true)
+        })
+        cy.get('[test-id="Further_information_Additional_details6_key"]').should('contain.text', 'Is your school linked to a diocese?')
+        cy.get('[test-id="Further_information_Additional_details6_value"]').should('contain.text', 'Yes')
+        cy.fixture('cath121-body.json').as('userData').then((userData) => {
+            expect(userData.data.applyingSchools[0].schoolFaithSchool).to.eq(true)
+        })
+        cy.get('[test-id="Further_information_Additional_details7_key"]').should('contain.text', 'Is your school part of a federation?')
+        cy.get('[test-id="Further_information_Additional_details7_value"]').should('contain.text', 'Yes')
+        cy.fixture('cath121-body.json').as('userData').then((userData) => {
+            expect(userData.data.applyingSchools[0].schoolIsPartOfFederation).to.eq(true)
+        })
+        cy.get('[test-id="Further_information_Additional_details8_key"]').should('contain.text', 'Is the school supported by a foundation, trust or other body (e.g. parish council) that appoints foundation governors?')
+        cy.get('[test-id="Further_information_Additional_details8_value"]').should('contain.text', 'Yes')
+        cy.fixture('cath121-body.json').as('userData').then((userData) => {
+            expect(userData.data.applyingSchools[0].schoolIsSupportedByFoundation).to.eq(true)
+        })
+        //"schoolSupportedFoundationBodyName": "Sed eius et." - THIS IS NOT INCLUDED ON THE FRONTEND
+        cy.get('[test-id="Further_information_Additional_details9_key"]').should('contain.text', 'Does the school currently have an exemption from providing broadly Christian collective worship issued by the local Standing Committee on Religious Education (SACRE)?')
+        cy.get('[test-id="Further_information_Additional_details9_value"]').should('contain.text', 'Yes')
+        cy.fixture('cath121-body.json').as('userData').then((userData) => {
+            expect(userData.data.applyingSchools[0].schoolHasSACREException).to.eq(true)
+        })
+        cy.get('[test-id="Further_information_Additional_details10_key"]').should('contain.text', 'Please provide a list of your main feeder schools')
+        cy.get('[test-id="Further_information_Additional_details10_value"]').should('contain.text', dataAppSch.schoolAdFeederSchools)
+        // 'Upload evidence' IS NOT BE COVERED IN THIS TEST AS THESE ARE UPLOAD LINKS- [test-id="Further_information_Additional_details11_key"]
```

Comment:
will be added as part of ticket 83618 - 'Add follow up answers to questions on application form'

Reply:
Notes added to script.